packages/actor-http-native/test/ActorHttpNative-test.ts

```diff
@@ -3,11 +3,40 @@ import * as url from 'url';
 import * as zlib from 'zlib';
 import { ActorHttp } from '@comunica/bus-http';
 import { Bus } from '@comunica/core';
-import { ActorHttpNative } from '../lib/ActorHttpNative';
+import { ActorHttpNative, processHttpResponse } from '../lib/ActorHttpNative';
 import Requester from '../lib/Requester';
 
+const http = require('http');
 const arrayifyStream = require('arrayify-stream');
 const mockSetup = require('./__mocks__/follow-redirects').mockSetup;
+const raw = [
```

Comment:
Only one test seems to be using this field (and `options`), can we move them in the scope of this tests?

The test themselves look great!